apis/scheduling/config/types.go

```diff
@@ -113,3 +113,13 @@ const (
 	// NUMADistributeEvenly indicates that evenly distribute CPUs across NUMA Nodes.
 	NUMADistributeEvenly NUMAAllocateStrategy = "DistributeEvenly"
 )
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ReservationArgs holds arguments used to configure the Reservation plugin.
+type ReservationArgs struct {
```

Comment:
A type with the same name also needs to be defined in `config/v1beta2`.

Reply:
fixed